a2/src/model/questionModel.js

```diff
@@ -0,0 +1,98 @@
+import {EventEmitter} from "events";
+
+class QuestionModel extends EventEmitter{
+    constructor(){
+        super();
+        this.state = {
+            questions: [{
+                id:0,
+                username: "u1",
+                title: "Question 1",
+                text: "text1",
+                creationDate: 2017,
+                tags: "tag1 tag2 tag3",
```

Comment:
Would have been better if you stored the tags as a list of strings instead of a single string.